bool img
---
sensor_msgs/Image cap_image 
